.github/workflows/operate-ci-test-reusable.yml

```diff
@@ -53,7 +53,7 @@ env:
 jobs:
   integration-tests:
     name: ${{ inputs.test-type }}
-    timeout-minutes: 20
+    timeout-minutes: 40
```

Comment:
This GHA workflow is also part of required status checks so timeout increases must be avoided - how does adding Optimize affect the Operate build?

Reply:
Some of the Optimize modules are taking a lot of time to build, compare to the other modules in this repo. Internally, we decided this is a problem we can solve later, and that we could keep the bigger timeouts for this iteration.

The reason why I increased those timeouts is because otherwise the workflows fail. Will talk to @RomanJRW and @OmranAbazid about possible solutions, if these must be avoided at all costs.

Thanks